backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -72,8 +74,16 @@ export class TournamentTree {
     return [...this.tournamentTreeArray];
   }
 
-  public getMatchesQueueReadyToBegin(): FightingTeamsGroup[] {
-    return this.tournamentTreeArray.filter(({ firstTeam, secondTeam }) => firstTeam && secondTeam);
+  public getMatchesQueueReadyToStart(): MatchReadyToStart[] {
+    return this.tournamentTreeArray
+      .filter(({ firstTeam, secondTeam }) => firstTeam && secondTeam)
```

Comment:
Nie powinno być jeszcze filtrowania, że mecz nie wystartował?

Reply:
Przegapiłem to pole... 🤦

Reply:
Drzewko nie przechowuje takich danych czy mecz wystartował - i ich nie potrzebuje.
Posiada tylko dane czy mecz się zakończył.

Reply:
@nowakprojects czy w takim razie chcemy, żeby drzewko miało taką informację? bo na tą chwilę mamy tylko flagę `isMatchFinished`

Reply:
Nie wiem, myślałem, że ma. A jak nie ma, no to wychodzi, że chyba nie potrzebuje. Ważne, żeby działało :)